.gitlab/deploy_dca.yml

```diff
@@ -10,6 +10,8 @@ include:
   extends: .docker_publish_job_definition
   stage: deploy_dca
   dependencies: []
+  needs:
```

Comment:
I'm surprised this works: `In GitLab 12.6 and later, you can’t combine the dependencies keyword with needs.`

I think you should replace `dependencies: []` with `needs: artifacts: false` (https://docs.gitlab.com/ee/ci/yaml/#needsartifacts)